### ISSUE TITLE
Projekt ded?

### ISSUE THREAD

**Issue by AyHa1810** [AOF-Dev/MCinaBox#1005]
Is the projekt ded? Oh c'mon devs, you can't just lost the hope for it bruh :/
Or even if you didn't, just comeback.

**Comment by SevenIsNotNumber** [comment]
> Is the projekt ded? Oh c'mon devs, you can't just lost the hope for it bruh :/
> Or even if you didn't, just comeback.

Yeah. Mcinabox must comeback to live. It rarely got an update

**Comment by LegacyGamerHD** [comment]
probely most of guys are in study stuff rn

**Comment by artdeell** [comment]
Remember the time when the devs vanished in the past?

**Comment by SevenIsNotNumber** [comment]
> Is the projekt ded? Oh c'mon devs, you can't just lost the hope for it bruh :/
> Or even if you didn't, just comeback.

Sad but its confirmed that longjunyu left the team

**Comment by AyHa1810** [comment]
Damn
Well I'm also having finals and I'm studing, and still I'm doing things on my pc and stuff

**Comment by LegacyGamerHD** [comment]
sorry guys i guesh go pojav for now wo knows maby some of will still work on it time wil tell als because longjunyu  did most of th e work i dont think its gone be anyting good so what now @CosineMath  your only one with any permision

**Comment by AyHa1810** [comment]
> sorry guys i guesh go pojav for now wo knows maby some of will still work on it time wil tell als because longjunyu did most of th e work i dont think its gone be anyting good so what now @CosineMath your only one with any permision

God your engrisu sounds like child lmfao XD

**Comment by SevenIsNotNumber** [comment]
> > sorry guys i guesh go pojav for now wo knows maby some of will still work on it time wil tell als because longjunyu did most of th e work i dont think its gone be anyting good so what now @CosineMath your only one with any permision
> 
> God your engrisu sounds like child lmfao XD


Hmmmm

**Comment by SevenIsNotNumber** [comment]
~~its becoz of that freaking phone~~

**Comment by AyHa1810** [comment]
but why seven is not a number?

**Comment by SevenIsNotNumber** [comment]
> but why seven is not a number?

Idk what my past self is thinking

**Comment by CosineMath** [comment]
Sorry to tell all of you that Long abandoned this project finally. It will reach end of life if no active developer continue to maintain. So anyone who want to try?

**Comment by CosineMath** [comment]
It is Long's decision and of course I cannot change his mind.

**Comment by Beau2222** [comment]
> It is Long's decision and of course I cannot change his mind.



> Sorry to tell all of you that Long abandoned this project finally. It will reach end of life if no active developer continue to maintain. So anyone who want to try?

Hmmmm i am learning how to develop things soooo maybe?

**Comment by NeonzGamingYT** [comment]
> Sorry to tell all of you that Long abandoned this project finally. It will reach end of life if no active developer continue to maintain. So anyone who want to try?

I like to try but i dont know how to code so yeah. I miss mcinabox. I want to alive back! 😢

**Comment by AyHa1810** [comment]
> Sorry to tell all of you that Long abandoned this project finally. It will reach end of life if no active developer continue to maintain. So anyone who want to try?

I might try from scratch, but I don't have any knowledge of programming. :/

**Comment by NeonzGamingYT** [comment]
> > Sorry to tell all of you that Long abandoned this project finally. It will reach end of life if no active developer continue to maintain. So anyone who want to try?
> 
> I might try from scratch, but I don't have any knowledge of programming. :/

Me too!

**Comment by LegacyGamerHD** [comment]
I can try idk if i can maitaine if can do

**Comment by NeonzGamingYT** [comment]
I think MCinabox needs to build from scratch again

**Comment by NeonzGamingYT** [comment]
Look what i did
![Screenshot_20210318_102211](https://user-images.githubusercontent.com/70279199/111563971-f0a31180-87d3-11eb-9983-c9856895a3d3.jpg)
Bruh

**Comment by AyHa1810** [comment]
> I can try idk if i can maitaine if can do

Your english will fuck up the project lol
im sorry

**Comment by AyHa1810** [comment]
wait, can @CosineMath save the project?

nah i dont think he's interested in it.

**Comment by NeonzGamingYT** [comment]
> wait, can @CosineMath save the project?
> 
> nah i dont think he's interested in it.

You and me can continue this project if we know coding. But i  try on apk editor

**Comment by AyHa1810** [comment]
What about @FrankOu2001? Ya here or ya work for pojav?

**Comment by LegacyGamerHD** [comment]
> > I can try idk if i can maitaine if can do
> 
> Your english will fuck up the project lol
> im sorry

And thnx but your English is not 100% to

**Comment by LegacyGamerHD** [comment]
O yeah I don't have work here as am already located pojav

**Comment by SevenIsNotNumber** [comment]
> > > I can try idk if i can maitaine if can do
> > 
> > 
> > Your english will fuck up the project lol
> > im sorry
> 
> And thnx but your English is not 100% to

Lol.. True.

**Comment by AyHa1810** [comment]
So ei tAip LaIk DiS oN pArPoUsE HaHa

**Comment by AyHa1810** [comment]
> > wait, can @CosineMath save the project?
> > nah i dont think he's interested in it.
> 
> You and me can continue this project if we know coding. But i try on apk editor

apk editor? You can't make an app work just by simply editing the installation package lol

**Comment by SevenIsNotNumber** [comment]
> > > wait, can @CosineMath save the project?
> > > nah i dont think he's interested in it.
> > 
> > 
> > You and me can continue this project if we know coding. But i try on apk editor
> 
> apk editor? You can't make an app work just by simply editing the installation package lol

Lol yeah

**Comment by NeonzGamingYT** [comment]
It doesn't matter bruh 😒

**Comment by SevenIsNotNumber** [comment]
R.I.P mchinabox. 
Mcinabye

**Comment by TheSafeBoi** [comment]
Bruh Mcinabox and Pojavlauncher is still not updating

**Comment by SevenIsNotNumber** [comment]
> Bruh Mcinabox and Pojavlauncher is still not updating

Actually there's alot happening on pojav. Theyre making zink work

**Comment by SevenIsNotNumber** [comment]
If it does, newest snapshots and shaders will work

**Comment by AyHa1810** [comment]
maybe @friendlyfredward can ask for help in his yt channel :P

**Comment by artdeell** [comment]
> Bruh Mcinabox and Pojavlauncher is still not updating

@TheSafeBoi we do update, but only in actions or on google play

**Comment by LegacyGamerHD** [comment]
> My broken english is for fun, not like I don't know english for real. Haha

can you just stop i know english to do its fucking hard typing on keyboard unlike auto corection your using

**Comment by LegacyGamerHD** [comment]
am done am leaving this f stupid repo

**Comment by AyHa1810** [comment]
whoa whoa calm down calm down im sorry

**Comment by AyHa1810** [comment]
i really suck at jokes and shit

**Comment by AyHa1810** [comment]
> > My broken english is for fun, not like I don't know english for real. Haha
> 
> can you just stop i know english to do its fucking hard typing on keyboard unlike auto corection your using

also wait, auto correction? My good old Hackers keyboard doesn't do that here
![Screenshot_2021-03-23-12-35-22-09](https://user-images.githubusercontent.com/63386848/112107096-8a076480-8bd4-11eb-8330-e9e6404d5653.jpg)

**Comment by SevenIsNotNumber** [comment]
Its been a month since mcinabox last update.

**Comment by TheSafeBoi** [comment]
æ

**Comment by SamSam759** [comment]
**"MCinabox"** is discontinued go pojav for now.

**Comment by SamSam759** [comment]
> am done am leaving this f stupid repo

Hey I saw you on the pojavlaucher github group lol

**Comment by NeonzGamingYT** [comment]
I still remember the past memory when mcinabox was so popular

**Comment by AyHa1810** [comment]
Hmm no one's gonna make it alive? damn

**Comment by FreddyFeuerstein** [comment]
If it was written in python, i would 100% continue it. But its java and there are only two things on earth i hate: Node.js and Java.

**Comment by AyderProgramic** [comment]
> Если бы он был написан на python, я бы на 100% продолжил его. Но его java и есть только две вещи на земле, которые я ненавижу: Node.js и Java.

Please come back the project needs you😭

**Comment by AyderProgramic** [comment]
Please come back the project needs you😭

**Comment by SavagePy10** [comment]
> If it was written in python, i would 100% continue it. But its java and there are only two things on earth i hate: Node.js and Java.

U can create your own launcher with VOC and python. U can access android api and stuffs like that.

**Comment by TheSafeBoi** [comment]
Rtpi

**Comment by sc-1123** [comment]
Um......
Even if this project died, I won't use pojav because pojav behaved bad on my device.
I'm currently learning Android programming, and if I can, I'd like to add support for Fabric to this project.

**Comment by sc-1123** [comment]
Um......
Even if this project died, I won't use pojav because pojav behaved bad on my device.
I'm currently learning Android programming, and if I can, I'd like to add support for Fabric to this project.

**Comment by AyderProgramic** [comment]
What?

**Comment by AyderProgramic** [comment]
какой смысл дублировать

**Comment by WuJiayiUbi** [comment]
mc is going to java 16, guess this means a big change to this proj

**Comment by kwvh** [comment]
> 对不起，伙计们，我现在要去 pojav，我知道有些人仍然会继续工作，时间会告诉其他人，因为 longjunyu 做了大部分工作，我认为它已经没有任何好处了，所以现在怎么办 @CosineMath 你唯一有任何许可的人

那个软件我也玩过，优化太少了  但容易闪退 在加了很多模组的情况下  但是在本软件是很少的情况 也希望你在那个软件的基础上留下一点 mcinabox小特色  比如自寻桌面配色 还是喜欢 mcinabox配色一点 也希望你在那个软件完成之后，继续维护本软件 希望如此

**Comment by Nishantsingh123** [comment]
Maybe I think @longjunyu2 is back to AOF

**Comment by Ivyjunsy** [comment]
Is Pojavlauncher and mcinabox created using a phone?? 🤔

**Comment by FreddyFeuerstein** [comment]
> Is Pojavlauncher and mcinabox created using a phone?? 🤔

yes (this is a joke btw but could actually be used to compile it. i used pcphonevm do do this)
![Screenshot_20210805-102953](https://user-images.githubusercontent.com/55027065/128318671-a0cae632-864b-4b94-827e-c75fd9ea277b.jpg)

**Comment by Ivyjunsy** [comment]
Is that an app??

**Comment by iAmInActions** [comment]
> What??
> What app are you using???

pcphonevm. i made that a while ago.
you cna get it here (aarch64 only): http://mazegame.de/sites/projects/PcPhoneVM/index.html

**Comment by Ivyjunsy** [comment]
Oh ok

**Comment by Ivyjunsy** [comment]
Can i make it like that??
Like your's??

**Comment by iAmInActions** [comment]
> Can i make it like that??
> Like your's??

yes. sry for replying on my alt btw, im freddyfeuerstein. You can swap xfce for lxde but xfce is more up-to-date.
The screenshots are pretty old btw and all the features are already implemented.

**Comment by Ivyjunsy** [comment]
How can i make it like this???
![Uploading 128318671-a0cae632-864b-4b94-827e-c75fd9ea277b.jpg…]()

**Comment by Ivyjunsy** [comment]
How can i make it like this??? 
![128318671-a0cae632-864b-4b94-827e-c75fd9ea277b](https://user-images.githubusercontent.com/87071336/128395024-fae813a1-760a-4701-b686-ccf3cda11ca6.jpg)

**Comment by NeonzGamingYT** [comment]
> How can i make it like this???
> ![128318671-a0cae632-864b-4b94-827e-c75fd9ea277b](https://user-images.githubusercontent.com/87071336/128395024-fae813a1-760a-4701-b686-ccf3cda11ca6.jpg)
> Can you give me the instruction??
It something called like termux

**Comment by Ivyjunsy** [comment]
Ohh okay..
I'll try to learn how to code.. cuz it sounds fun..

**Comment by Ivyjunsy** [comment]
How did you guy's learn how to code??

**Comment by FreddyFeuerstein** [comment]
> How did you guys learn how to code??

I learned how to code bc my first pc was really slow and couldnt run the ubuntu desktop very well. My first project (fully done with nano and gcc in tty shells and links2 to research on the internet) was a tiny window manager that didn't need any big libraries and ran smooth in xvesa. I sadly lost the code for that project so I cannot share it and even if i still had it, i wouldnt bc it would probably brick someones pc bc its a mess that only ran on my pc.

Nowadays, I use IceWM or LXDE as my desktop and I found node.js and python to be my favourite languages.

**Comment by Ivyjunsy** [comment]
Did you took a learning course on coding??
Is there learning course for coding?? I really don't know.. 😅😅

**Comment by kwvh** [comment]
Ah this

在 2021-08-07 20:49:18，Ivyjunsy ***@***.***> 写道：


Did you took a learning course on coding??
Is there learning course for coding?? I really don't know.. 😅😅

**Comment by MTM828** [comment]
> How did you guys learn how to code??

Learn a simple interpreted language like Python at first, then go for a compiled language like C/C++. You can also try semi-compiled langs (like Java and Lua).
When learning a new lang, go for a simple tutorial that teaches u the basics, and then start making projects (games, scripts, tools etc.), you'll gain more experience while writing code.
Happy coding!

**Comment by AjGamingPHOfficial** [comment]
I miss this launcher

**Comment by TSaltedfishKing** [comment]
> I miss this launcher

me too

**Comment by AtlasGaming175** [comment]
Why mcinabox no update?

**Comment by iAmInActions** [comment]
> Why mcinabox no update?

because there is no one maintaining it atm. If you want you can make an update for it though.

**Comment by ankris812** [comment]
FINE, I'LL DO IT MYSELF
